gradle.properties

```diff
@@ -15,7 +15,7 @@
 #
 version=0.11.2
 ideVersion=PC-2021.2.3
-pythonPlugin=PythonCore:212.5457.59
+pythonPlugin=python-ce
```

Comment:
This change was needed to be able to run the new tests.

Without this, the `createLightFile()` function complained that there were multiple plugins trying to handle the Python file type. Since `PC-2021.2.3` includes the Community Edition Python plugin, maybe it was trying to load two plugins at once? In any case, I don't think there's any downside to this change.